PIPs/PIP-02.md

```diff
@@ -1,6 +1,12 @@
-| PIP               | Title                           | Description          | Author                        | Discussion | Status | Type                                     | Date                  |
-|-------------------|---------------------------------|----------------------|-------------------------------|------------|--------|------------------------------------------|-----------------------|
-| 2 | Proposal to distribute and decentralize staking power across the Polygon chain  | Proposal to increase the minimun commission. | Delroy Bosco | [Forum](https://forum.polygon.technology/t/pip-2-proposal-to-distribute-and-decentralize-staking-power-across-the-polygon-chain/219/11)  | Stagnant | Contracts | 2021-11-01
+---
+pip: 2
+title: Proposal to distribute and decentralize staking power across the Polygon chain
+description: Proposal to increase the minimun commission.
+author: Delroy Bosco
```

Comment:
I couldn't find a GitHub username for Delroy, so any future changes to this file will report an error on this line. Mostly just an annoyance.

Reply:
Thanks for pointing it out, I can find the username if needs be.